### PR TITLE
TAJO-1325: Invalid history cleaner timeout

### PR DESCRIPTION
**Pull request by jinossy**

### CODE REVIEW

File: tajo-core/src/main/java/org/apache/tajo/querymaster/QueryMaster.java [hunk 1]
```diff
@@ -525,7 +524,13 @@ private void cleanExpiredFinishedQueryMasterTask(long expireTime) {
       synchronized(finishedQueryMasterTasks) {
         List<QueryId> expiredQueryIds = new ArrayList<QueryId>();
         for(Map.Entry<QueryId, QueryMasterTask> entry: finishedQueryMasterTasks.entrySet()) {
-          if(entry.getValue().getStartTime() < expireTime) {
+          long finishedTime = entry.getValue().getStartTime();
```
Comment (jihoonson) [inline tajo-core/src/main/java/org/apache/tajo/querymaster/QueryMaster.java]: Would you leave some comments?

File: tajo-core/src/main/java/org/apache/tajo/worker/TaskRunnerManager.java [hunk 2]
```diff
@@ -240,7 +240,9 @@ private void cleanExpiredFinishedQueryMasterTask(long expireTime) {
       synchronized(taskRunnerHistoryMap) {
         List<String> expiredIds = new ArrayList<String>();
         for(Map.Entry<String, TaskRunnerHistory> entry: taskRunnerHistoryMap.entrySet()) {
-          if(entry.getValue().getStartTime() > expireTime) {
+
+          long finishedTime = Math.max(entry.getValue().getStartTime(), entry.getValue().getFinishTime());
```
Comment (jihoonson) [inline tajo-core/src/main/java/org/apache/tajo/worker/TaskRunnerManager.java]: Would you leave some comments?

**Comment by jihoonson** [comment]
+1 LGTM.
Please add some explanations before you commit.

File: tajo-core/src/main/java/org/apache/tajo/worker/TaskRunnerManager.java (hunk 2 above)
Comment (jinossy) [inline tajo-core/src/main/java/org/apache/tajo/worker/TaskRunnerManager.java]: Sure!

**Pull request closed without merging**